admin/class-openedx-woocommerce-plugin-admin.php

```diff
@@ -269,7 +269,11 @@ public function add_custom_column_order_items() {
 	public function add_admin_order_item_values( $_product, $item, $item_id = null ) {
 
 		// Check if the product has a non-empty "_course_id" metadata.
-		$_course_id = get_post_meta( $_product->get_id(), '_course_id', true );
+		$_course_id = '';
```

Comment:
I know this is old but can you rename the variable to remove the first underscore "_" to change from $_course_id to $course_id

Reply:
Done.